lib/zlib.js

```diff
@@ -347,7 +347,7 @@ function Zlib(opts, mode) {
 
   var self = this;
   this._hadError = false;
-  this._handle.onerror = function(message, errno) {
+  this._handle.onerror = (message, errno) => {
```

Comment:
I wouldn't change this one to an arrow function. There's really no point in doing so here. I would just add a name to the existing function